### ISSUE TITLE
allow column presets for admin role

### ISSUE THREAD

**Issue by natinati8520** [hasura/graphql-engine#6519]
currently, the admin role has no possible way to assign needed column presets, such as the other roles have.
add that option.
In the screenshot below, all the circles permissions aren't clickable, and therefore there's no way to assign a column preset.

![Inkedscreenshot_LI](https://user-images.githubusercontent.com/58391323/106892490-cf3f0800-66f4-11eb-8af4-3d1c8d970ad8.jpg)

**Comment by natinati8520** [comment]
bump :(

**Comment by tirumaraiselvan** [comment]
Not sure if we will allow this, since `admin` by definition has full control. 

Although, we can enhance the UI capability to be able to copy admin permissions to any role, resource combination and then you can set the column presets for that role.

**Comment by mrkpatchaa** [comment]
Light be useful for exemple if you want to set user id on a table, with session information without sending it explicitly. 
I just run into this use case.

**Comment by DafyddLlyr** [comment]
Agreed - enabling this would be very helpful, I have essentially the same use case as @mrkpatchaa

**Comment by RoniFinTech** [comment]
this feature will be helpful.
I have other roles that will preset "author_id"... however, under the Admin role, I need to provide that field.

**Comment by Kostanos** [comment]
Any chance for this feature to be implemented?

I have updatedBy, createdBy fields. Would be great to populate those values for admin role too.

**Comment by pscanlon1** [comment]
bump

**Comment by likt0r** [comment]
bump

**Comment by agrandin0** [comment]
This feature will be very helpful...we've got a created_by field with a column preset for other role...
